pkg/util/cluster/cluster.go

```diff
@@ -64,6 +64,7 @@ type Cluster struct {
 	openshiftclustersv20220904        redhatopenshift20220904.OpenShiftClustersClient
 	securitygroups                    network.SecurityGroupsClient
 	subnets                           network.SubnetsClient
+	vnets                             network.VirtualNetworksClient
```

Comment:
This field is only used in the _New_ function, so it is really never used. Can we removed it for now?

Reply:
Yes I will remove it, I added it for the previous changes but now it is not required.